builtin/logical/aws/secret_access_keys.go

```diff
@@ -182,6 +182,53 @@ func (b *backend) getFederationToken(ctx context.Context, s logical.Storage,
 	return resp, nil
 }
 
+// NOTE: Getting session tokens with or without MFA/TOTP has behavior that can cause confusion.
+// When an AWS IAM user has a policy attached requireing an MFA code by use of "aws:MultiFactorAuthPresent": "true",
```

Comment:
```suggestion
// When an AWS IAM user has a policy attached requiring a MFA code by use of "aws:MultiFactorAuthPresent": "true",
```

Reply:
I believe "an" is correct since M is spoken like "em" audibly.